### ISSUE TITLE
ICICI bank home page! Previous button is not label properly 

### ISSUE THREAD

**Issue by nsingh2001** [nsingh2001/bicici#5]
UAT 2022-04
Windows 10
Chrome Version 100.0.4896.88 (Official Build) (64-bit)

NVDA 2021.3.1
Repro Steps
1. Open the URL  https://www.icicibank.com/
2.after tab 19 time there is previous button 

Expected Result the previous button should be announce the purpose properly
Actual Result .screen reader announce only previous button 
WCAG Guideline
1. https://www.w3.org/TR/WCAG21/#name-role-value
